src/recipes.jl

```diff
@@ -550,7 +550,7 @@ Plots.@deps stepbins path
 function _auto_binning_nbins{N}(vs::NTuple{N,AbstractVector}, dim::Integer; mode::Symbol = :auto)
     _cl(x) = max(ceil(Int, x), 1)
```

Comment:
@oschulz the `ceil(Int, x)` but will fail if x is ever NaN. I assume this is not a problem? (just checking)